### DISCUSSION TITLE
Problem with appending objects to a georeferenced BlenderGIS 2.2.5 model.

### DISCUSSION THREAD

**Discussion by blendermagic** [domlysz/BlenderGIS#560] [Q&A] [answered]
I imported a GeoTIFF as DEM raw data with build faces (GIS/Import/Georeferenced raster) into Blender 2.90.0 to produce a terrain model. I normally do this in the Layout Tab which works well. There is a problem, however, if I then try to append external objects to that model. Specifically, the appended objects come in, but are **only** visible in the Modelling Tab. The problem reverses when starting in the Modelling Tab.

I do not understand the reason for this separation! Is the resultant GeoTIFF model a true mesh, or a point cloud object that Blender does not like? Any thoughts?

**Comment by blendermagic** [comment, marked as answer]
Solved! It turns out that the model origin was way off! The solution was to reset it, and to move the model to 0,0,0. I’m curious to know what determines the location of the origin. Is it the original GeoTIFF? I extracted it from a larger tile which may have caused the problem.

By the way….is there a limit to the number of images that BlenderGIS is able to mosaic? I am unable to export anything much above 10,000!

**Comment by domlysz** [comment]
More informations about georeferencing management can be found here https://github.com/domlysz/BlenderGIS/wiki/Georeferencing-management

For instance, when you import a geodata (geotiff, shapefile, basemap...) for the first time, the resulting object is placed at the scene origin and the geographic coordinate of the origin is stored by the addon. If you import another geodata, the new object will be placed accordingly to the geographic coordinate of the scene origin, so if you second import is far away the first one or if they don't share the same projection, then your second import can be placed far aways the origin.

For the second question, if you talking about the basemaps operator and the stiching process of tiles, then you can refer to the discussion #556

**Reply by blendermagic** [reply]
Thank you!
